src/components/Tooltip/__tests__/__snapshots__/Tooltip.spec.js.snap

```diff
@@ -46,10 +46,10 @@ exports[`Tooltip expect overlay to appear correctly when shown 1`] = `
                 style="position: absolute; z-index: 99999;"
               >
                 <div
-                  style="opacity: 0;"
+                  style="opacity: 0; transition: all 400ms ease-in-out;"
                 >
                   <div
-                    style="position: absolute; display: block; height: 12px; width: 12px; background-color: rgb(255, 255, 255); border-left: 1px solid #BDBDBD; border-top: 1px solid #BDBDBD; left: -6px; top: -7px;"
+                    style="position: absolute; display: block; height: 12px; width: 12px; background-color: rgb(255, 255, 255); transform: rotate(-45deg); border-left: 1px solid #BDBDBD; border-top: 1px solid #BDBDBD; left: -6px; top: -7px;"
```

Comment:
>transform: rotate(-45deg);

I'm not sure if we intend to add a rotate here? but it wasn't there before. and 45degree icons might look weird?

Reply:
is this a snapshot of one step from an animation?